### PR TITLE
feat: added support for label for Address component

### PR DESCRIPTION
**Pull request by adamhaeger**
## Description

Address component now supports a custom label if you add config to ie:

```
     {
        "id": "adresse",
        "type": "Address",
        "textResourceBindings": {
          "title": "Adresse"
        },
        "dataModelBindings": {
          "address": "Adresse.Gateadresse_æøå",
          "zipCode": "Adresse.Postnr",
          "postPlace": "Adresse.Poststed"
        },
        "simplified": true,
        "readOnly": false,
        "required": false
      },
```

## Related Issue(s)

- closes #{36}

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [x] Unit test(s) have been added/updated
  - [ ] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [ ] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [ ] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [ ] No functionality has been changed/added, so no documentation is needed
  - [ ] I will do that later/have created an issue
  
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [ ] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [ ] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [ ] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

**Comment by olemartinorg** [comment]
Nice! :raised_hands: Maybe you could look into adding bindings for the other components in there as well? (zip, postPlace, etc).

**Review by bjosttveit — COMMENTED** [review]
Also, dont forget to update the config so that the types and schemas are generated correctly. `Address/config.ts`, you can look at some other ones for inspiration

**Comment by adamhaeger** [comment]
> Also, dont forget to update the config so that the types and schemas are generated correctly. `Address/config.ts`, you can look at some other ones for inspiration

Types have been added, as well as support for labels for the other fields :)

File: src/layout/Address/config.ts [hunk 1]
```diff
@@ -15,7 +15,35 @@ export const Config = new CG.component({
     new CG.trb({
       name: 'title',
       title: 'Title from Summary',
```
Comment (bjosttveit) [inline src/layout/Address/config.ts:17]: This is no longer just for summary, right?

Comment (adamhaeger) [reply src/layout/Address/config.ts]: Correct, fixed :)

**Review by bjosttveit — APPROVED** [review]
## Very good 🥇

**Pull request merged**